### PR TITLE
inochi-session: 0.8.4 -> 0.8.7, fix aarch64-linux; inochi-creator: 0.8.5 -> 0.8.6

### PR DESCRIPTION
**Pull request by RossComputerGuy**
## Things done

Now we can do VTuber stuff on Ampere Altra and Asahi NixOS!

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by TomaSajt** [comment]
Seems fine, can't test it ATM.

Is this patch from somewhere, or did you come up with it for this PR?

**Comment by RossComputerGuy** [comment]
Like the comment mentions, it comes from upstream.

**Comment by RossComputerGuy** [comment]
> Just noticed: PR and commit message says 0.8.5 instead of 0.8.4

Typo

> Also, would it not make sense to also update inochi-creator?

Does creator use Lua as well and could have the same issue so a similar patch would have to be made?

**Comment by TomaSajt** [comment]
> > Also, would it not make sense to also update inochi-creator?
> 
> Does creator use Lua as well and could have the same issue so a similar patch would have to be made?

Nope, just a version bump. But if you think it's better, we can keep it in another PR.

**Comment by RossComputerGuy** [comment]
Ok, I got creator updated too.

File: pkgs/applications/misc/inochi2d/generic.nix [hunk 1]
```diff
@@ -99,6 +100,7 @@ buildDubPackage (
       . gentl.sh
 
       # Use the fake git to generate version info
+      dub build --config=update-version
       dub build --skip-registry=all --compiler=ldc2 --build=release --config=meta
```
Comment (TomaSajt) [inline pkgs/applications/misc/inochi2d/generic.nix:104]:
```suggestion
      dub build --skip-registry=all --compiler=ldc2 --build=release --config=update-version
```
The meta task is no longer necessary, it's windows-only logic.

Comment (RossComputerGuy) [reply pkgs/applications/misc/inochi2d/generic.nix]: Done

**Review by TomaSajt — APPROVED** [review]
LGTM

**Pull request merged**